prepare_data/src/main/groovy/org/orbisgis/bdtopo/BDTopoGISLayers.groovy

```diff
@@ -64,18 +64,18 @@ static IProcess importPreprocess(){
                 logger.info('Executing the importPreprocess.sql script')
                 def uuid = UUID.randomUUID().toString().replaceAll('-', '_')
                 def tmpIris = 'TMP_IRIS_' + uuid
-                def zone = 'ZONE_' + uuid
+                def zone = 'ZONE'
```

Comment:
Why delete the uuid value?

Reply:
The uuid was sometimes used also for the output Tables while it is only useful for intermediate tables.